### PR TITLE
Bug fixes in User Authentication Work

### LINKED ISSUE #81: Inconsistent routing in the dashboard?
Moving over to either of `localhost:8000/#/` or `localhost:8000/#/dashboard` gives a 404.

### Expected behaviour

According to our umi routing configuration, it should automatically redirect to `http://localhost:8000/#/dashboard/dashboard` which seems to be a valid route. 

### Demo

Screenshot 2020-08-17 at 4 09 49 PM

### PR DESCRIPTION
**Pull request by anishaswain**
Fixes #81 
Fixes `warning: <avatarIcon /> is using incorrect casing.`

### CODE REVIEW

**Comment by anishaswain** [comment]
@aquibbaig let me know if the PR resolves the issues in #81

**Review by aquibbaig — APPROVED** [review]
LGTM :rocket:

File: src/components/GlobalHeader/index.js [hunk 1]
```diff
@@ -69,11 +69,10 @@ class GlobalHeader extends Component {
   };
 
   logoutSession = () => {
-    const { dispatch } = this.props;
+    const { dispatch, auth } = this.props;
```
Comment (gurbirkalsi) [inline src/components/GlobalHeader/index.js]: The `auth` namespace is declared but its value is never read. Can we consider removing this reference?

**Review by gurbirkalsi — CHANGES_REQUESTED** [review]
Great work @Anisha1234, just one small fix before we merge!

**Review by gurbirkalsi — APPROVED** [review]

**Pull request merged**